zboxcore/zboxutil/http.go

```diff
@@ -457,9 +476,8 @@ func MakeSCRestAPICall(scAddress string, relativePath string, params map[string]
 			q.Add(k, v)
 		}
 		urlObj.RawQuery = q.Encode()
-		client := &http.Client{Transport: transport}
 
-		response, err := client.Get(urlObj.String())
+		response, err := util.NewHTTPNetContext().Get(urlObj.String(), &http.Client{Transport: transport})
```

Comment:
You might like to cosider  https://github.com/0chain/gosdk/issues/103.

Reply:
`devserver` part has been dropped, and follow it on  follow it on 0chain/blobber#145